### ISSUE TITLE
JDK11 nightlies failing due to AskSupport error

### ISSUE THREAD

**Issue by raboof** [akka/akka-http#2652]
The nightly builds on jdk11 are failing with:

```
Symbol 'type akka.pattern.AskSupport' is missing from the classpath
```

This sounds like a similar issue to https://github.com/akka/akka/issues/26503, which should be fixed when sbt 1.3.0 is released and we can switch to it...

**Comment by johanandren** [comment]
But why does it happen on builds that do a clean first?

**Comment by jrudolph** [comment]
Yeah, I guess the reason is that we are doing a `clean` but that might not propagate to source dependency modules.

**Comment by jrudolph** [comment]
> Yeah, I guess the reason is that we are doing a `clean` but that might not propagate to source dependency modules.

No, that's not it. The jenkins task manually deletes the complete checkout of akka before every build. It seems instead that `akka-actor` is first build successfully but some files want to be rebuild shortly after when the next task is run.

**Comment by jrudolph** [comment]
It can be reproduced with Scala 2.12.8 but not with 2.12.9?!?

**Comment by jrudolph** [comment]
Don't want to spend any more time on this. Disabled JDK 11 builds against master for now. Hopefully it's going to be fixed once we are able to switch to sbt 1.3.0.

**Comment by jrudolph** [comment]
This has since been fixed. The latest nightlies do build against master on JDK 11 again: https://jenkins.akka.io:8498/job/akka-http-nightly/
